### PR TITLE
re-enable cluster test

### PR DESCRIPTION
**Pull request by davejrt**
This re-enabled the tests previously disabled due to bad migrations. The new migrator has been [added](https://github.com/sourcegraph/deploy-sourcegraph/commit/8fd0f29b90200a79859553b9be0c0c17ee327e0d) to the deploy-Sourcegraph repo so we should have a reliable test suite here now. 

closes https://github.com/sourcegraph/sourcegraph/issues/25977

### CODE REVIEW

**Review by bobheadxi — APPROVED** [review]
woo!

there seems to be an emtpy `dev/ci/integration/cluster/run.sh` - is that intentional?

File: enterprise/dev/ci/internal/ci/operations.go [hunk 1]
```diff
@@ -577,22 +577,22 @@ func testUpgrade(candidateTag, minimumUpgradeableVersion string) operations.Oper
 }
 
 // Flaky deployment. See https://github.com/sourcegraph/sourcegraph/issues/25977
```
Comment (bobheadxi) [inline enterprise/dev/ci/internal/ci/operations.go:579]: Outdated docstring?

```suggestion
```

File: enterprise/dev/ci/internal/ci/pipeline.go [hunk 2]
```diff
@@ -244,7 +244,7 @@ func GeneratePipeline(c Config) (*bk.Pipeline, error) {
 			serverE2E(c.candidateImageTag()),
 			serverQA(c.candidateImageTag()),
 			// Flaky deployment. See https://github.com/sourcegraph/sourcegraph/issues/25977
```
Comment (bobheadxi) [inline enterprise/dev/ci/internal/ci/pipeline.go:246]:
```suggestion
```

**Comment by davejrt** [comment]
> woo!
> 
> there seems to be an emtpy `dev/ci/integration/cluster/run.sh` - is that intentional?

I think it's because of a permissions change

`00644 → 100755`

**Comment by bobheadxi** [comment]
> I think it's because of a permissions change

Ah you're right! The GitHub diff shows an empty file but it's actually a no diff 😬 

image

**Review by bobheadxi — APPROVED** [review]

**Comment by davejrt** [comment]
> > I think it's because of a permissions change
> 
> Ah you're right! The GitHub diff shows an empty file but it's actually a no diff grimacing
> 
> image

Not super helpful in this case.

**Pull request merged**